FAQs.md

```diff
@@ -0,0 +1,121 @@
+
+1. **What is Jockey Music Player?**
+
+Jockey is a music player for Android based on Google's Material Design standards. The point of Jockey is to be a simple, lightweight media player with features designed for music enthusiasts and casual listeners alike. 
+
+
+2. **What are the features of jockey music player?**
+
+Jockey music player is designed with the essential features of a music player and they include;
+
+- playlist
+- theme customization
+- equalizer
+- sleep timer
+- search tool
+
+3. **How can I navigate between songs, artists, album, genres and playlists?**
+
+You can navigate to these pages simply by swiping left or right depending on the position.
+
+4. **How do I create a create a playlist?**
+
+To create a playlist on Jockey, navigate to the playlist tab by swiping left or right. On the playlist, tab click the plus (+) sign below and two options will pop up Smart Playlist and Playlist. Click the plus sign beside playlist, input playlist name and click on create.
+
+5. **What is smart playlist?**
+
+Smart playlist as the name implies is a playlist that automatically sorts and add music to the list according to customized rules or parameters set. 
+
+6. **Can I import and export playlists on Jockey music player?**
+
+No, playlists can neither be imported nor exported.
+
+7. **How can I add songs to a playlist?**
+
+Songs can be added to a playlist from any of the tabs. Songs, Artists, Albums, Genres. To add a song, simply click the 3 dots beside the song title and select the "Add to Playlist" option and select the desired playlist you wish the song to be added to.
+
+
+8. **How do I add a song to my favorite playlist?**
+
+Jockey player doesn't have the add favorite button. To add a song to a favorite playlist, tap on the vertical dots on a song then tap on add to playlist, and tap on favorite.
+
+
+9. **Can I add highlight multiple songs at once to add a playlist?**
+
+No, the highlight function is not yet implemented on Jockey music player. However, when an artist, album or genre containing multiple songs are added to a playlist all the songs contained are added.
+
+10. **Can I sort tracks?**
+
+Only tracks contained in a playlist can be sorted
+
+11. **Can I include or exclude folders from my music library?**
+
+Yes, this function is available on Jockey music player.
+
+
+12. **How can I include or exclude a folder from my music library?**
+
+- Access Jockey player settings by clicking the 3 dots on the top right side of the app and selecting settings. 
+- Scroll down to library and select either to include or exclude. 
+- Click the plus(+) button and click on the folder you wish to include or exclude then click select.
+
+13. **Do I have to manually scan new tracks into the music player?**
+
+No, Jockey music player automatically scans library for new tracks.
+
+14. **How do I remove an included or excluded folder from the list?**
+On the include/exclude list, click the 3 dots beside the folder and select the remove option.
```

Comment:
Seems like this should come before 13 so that it's grouped with the other include/exclude points.